### DISCUSSION TITLE
有考虑过添加 「大小写」键切换输入法吗？

### DISCUSSION THREAD

**Discussion by Attiv** [fcitx-contrib/fcitx5-macos#100] [General]
就像原生的那样。
不然每次更新完都得手动执行 `cd /Library/Input\ Methods/Fcitx5.app/Contents  && sudo /usr/libexec/PlistBuddy -c 'Add :TICapsLockLanguageSwitchCapable bool true' Info.plist`
更新完又经常忘输入，所以每次还要登出就很麻烦

**Comment by eagleoflqj** [comment]
于 https://github.com/fcitx-contrib/fcitx5-macos/commit/403d8af2b05c2ddd5af6320a8fbf05c1e054d5be 完成

**Comment by eagleoflqj** [comment]
自0.2.2起，由于小企鹅注册为英语输入法（[原因](https://fcitx-contrib.github.io/docs/faq.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E5%B0%8F%E4%BC%81%E9%B9%85%E6%B3%A8%E5%86%8C%E4%B8%BA%E8%8B%B1%E8%AF%AD%E8%BE%93%E5%85%A5%E6%B3%95)），此功能需要通过改键软件实现（[步骤](https://fcitx-contrib.github.io/docs/faq.html#%E6%88%91%E6%83%B3%E4%BD%BF%E7%94%A8-capslock-%E5%88%87%E6%8D%A2%E4%B8%AD%E8%8B%B1%E6%96%87)）
